---
layout: post
title: A modular, nondegenerate polyprotein scaffold for AFM
date: 2016-06-08
comments: false
archive: false
tags: single-molecule, force-spectroscopy, dna
research: true
link: http://dx.doi.org/10.1021/acs.biomac.6b00548
---

<p>
<b>Zackary N. Scholl</b>, Eric Josepshs, and Piotr E. Marszalek. "Modular, Nondegenerate Polyprotein Scaffolds for Atomic Force Spectroscopy." Biomacromoleulces (2016). <a href="http://dx.doi.org/10.1021/acs.biomac.6b00548">Link</a>
</p>
<p class="lead">We improved on earlier polyprotein designs to generate a new plasmid backbone, pEMI91, that resolves issues using tandem repeats using “shuffled” codons that can be easily sequenced. The plasmid for the polyprotein was designed to contain nine consecutive cassettes that would each translate to the I91 domain from Homo sapiens TTN gene titin. Each of these cassettes is separated by unique restriction sites for simple replacement of individual I91 cassettes with the DNA coding for a protein of interest, and by virtue of using “unique” I91-coding DNA sequences, a series of primers have been developed that can be used to sequence across the entire polyprotein. The polyprotein itself is flanked by a His-tag and a Strep-tag to allow for simplified purification of the expressed protein and possesses two C-terminal cysteine residues to aid in protein display.</p>

<!-- <div class="col-sm-12">
    <p align="center">
<img src="{{ "/assets/images/fe_detection.jpg" | prepend: site.url }}" alt="Force-spectroscopy resolves the origin of polydA stretching transitions" class="img-rounded img-responsive">
    </p>
</div>



<p>Automation can be accomplished using an algorithm that triages usable force–extension recordings quickly with positive and negative selection. We implemented an algorithm based on the windowed fast Fourier transform of force–extension traces that identifies peaks using force–extension regimes to correctly identify usable recordings from proteins composed of repeated domains. </p>



<p>This algorithm excels as a real-time diagnostic because it involves <30 ms computational time, has high sensitivity and specificity, and efficiently detects weak unfolding events. We used the statistics provided by the automated procedure to clearly demonstrate the properties of molecular adhesion and how these properties change with differences in the cantilever tip and protein functional groups and protein age.</p>
 -->



